src/ape_cache/_cli.py

```diff
@@ -0,0 +1,53 @@
+import click
+import pandas as pd
+
+from ape import networks
+from ape.cli import NetworkBoundCommand, network_option
+from ape.logging import logger
+from ape.utils import ManagerAccessMixin
+
+
+def get_engine():
+    return ManagerAccessMixin.query_manager.engines["cache"]
+
+
+@click.group(short_help="Query from caching database")
+def cli():
+    """
+    Command-line helper for managing query caching database.
+    """
+
+
+@cli.command(short_help="Initialize a new cache database")
+@network_option()
+def init(network):
+    provider = networks.get_provider_from_choice(network)
+    ecosystem_name = provider.network.ecosystem.name
+    network_name = provider.network.name
+
+    get_engine().init_database(ecosystem_name, network_name)
+    logger.info(f"Caching database initialized for {ecosystem_name}:{network_name}.")
+
+
+@cli.command(
+    cls=NetworkBoundCommand,
+    short_help="Call and print SQL statement to the cache database",
```

Comment:
Question: Does this CLI only apply to this plugin or will other non-SQL based engines also be trigger-able ere?
If so, I think we should refrain from saying `SQL` in the help texts if it potentially something else.